packages/lib-classifier/src/store/UserProjectPreferencesStore.js

```diff
@@ -64,10 +48,29 @@ const UserProjectPreferencesStore = types
       }
     }
 
-    function * fetchUPP (bearerToken, project, user) {
+    function * checkForUser () {
+      const { authClient } = getRoot(self)
+
+      try {
+        const bearerToken = yield getBearerToken(authClient)
+        const user = yield authClient.checkCurrent()
+        if (bearerToken && user) {
```

Comment:
If there's a user, there must be a valid bearer token, so checking both is redundant here?

Reply:
We need the user id for the query param and a defined bearer token for making the request authenticated. It does seem redundant, but I'm not totally trusting of the PJC auth client. I don't want to make the request if for some reason either of those are undefined.